code/modules/mob/living/carbon/carbon.dm

```diff
@@ -781,5 +781,12 @@
 
 	..()
 
-
+/mob/living/carbon/adjustToxLoss(amount, updating_health=1)
+	if(TOXINLOVER in dna.species.specflags) //damage becomes healing and healing becomes damage
+		amount = -amount
+		if(amount > 0)
+			blood_volume -= 5*amount
+		else
+			blood_volume -= amount
+	..()

```

Comment:
return ..()
(ideally all damage procs would return the damage done but not all of them do so yet)